### ISSUE TITLE
Cannot set environment variables

### ISSUE THREAD

**Issue by dciccale** [gimm/gulp-live-server#34]
hi, on version `0.0.7` i was able to set some environment variable before starting the server and it would work, but not now.

``` javascript
gulp.task('serve-dist', function () {
  process.env.NODE_ENV = 'production';
  g.liveServer(['./dist/server/app.js'], {}, false).start();
});
```

how can i handle this with the new version?

**Comment by dciccale** [comment]
ok, got it working. this is what i did

``` javascript
gulp.task('serve-dist', function () {
  var options = {env: process.env};
  options.env.NODE_ENV = 'production';
  g.liveServer(['./dist/server/app.js'], options, false).start();
});
```
